contracts/Vault.vy

```diff
@@ -420,6 +723,43 @@ def maxAvailableShares() -> uint256:
 
 @external
 def withdraw(_shares: uint256 = MAX_UINT256, _recipient: address = msg.sender) -> uint256:
+    """
+    @notice 
+        Withdraws the calling account's tokens from this Vault, redeeming
+        amount `_shares` for an appropriate amount of tokens.
+
+        Note that funds will be withdrawn from the least impactful Strategy
+        (the Strategy that will impact Vault gains the least by having funds
+        removed), then the next least impactful, and so on, until all `_shares`
+        have been redeemed for tokens.
```

Comment:
Maybe just say "see note on withdrawal queue"

Reply:
Will do

Reply:
Note - I'm assuming you mean `setWithdrawalQueue` since `withdrawalQueue` are comments, not NatSpec, and so wouldn't appear in any extracted documentation.